auth/src/main/java/org/aerogear/mobile/security/util/BuildReader.java

```diff
@@ -0,0 +1,80 @@
+package org.aerogear.mobile.security.util;
+
+import android.os.Build;
+
+/**
+ * Class to allow simpler testing of classes that use Build.
+ */
+public class BuildReader {
```

Comment:
is this class only used for testing? If this is the case should it be moved to the test package?

Reply:
This isn't meant to be here! 😅